src/index.ts

```diff
@@ -79,3 +73,17 @@ export const getKeypairFromEnvironment = (variableName: string) => {
   }
   return Keypair.fromSecretKey(decodedSecretKey);
 };
+
+export const addKeypairToEnvironment = (variableName: string) => {
+  const secretKeyString = process.env[variableName];
+  if (!secretKeyString) {
+    // Generate a new keypair if one doesn't exist and add it to a `.env` file
+    const keypair = Keypair.generate();
+    appendFileSync(
+      ".env",
+      `\n${variableName}=${JSON.stringify(Array.from(keypair.secretKey))}`,
```

Comment:
Not super important, but this (`JSON.stringify(Array.from(keypair.secretKey))`) is useful elsewhere so worth pulling out and exporting as `keypairToJSON()` or similar.